_i18n/en.yml

```diff
@@ -379,6 +379,9 @@ faq:
   adangernode: "Running a personal @node is the safest way to interact with the Monero network, because you are in full control and you don't need to rely on third parties. From a general point of view running a node is not dangerous, but keep in mind that your ISP can see you are running a Monero node."
   qdangerrnode: Is it dangerous to use a remote node? What's the data a node operator can get from me?
   adangerrnode: "It's always advisable, especially for privacy-conscious users, to use a personal node when transacting on the network to achieve the highest rate of privacy. Some people for convenience prefer to use @remote-node which are not under their control (public nodes). The convenience of not having to deal with a personal copy of the @blockchain comes at a cost: lessened privacy. A remote node operator is able to see from what IP address a transaction comes from (even if cannot see the recipient nor the amount) and in some extreme cases, can make attacks able to reduce your privacy. Some dangers can be mitigated by using remote nodes on the Tor or I2P networks or using a VPN."
+  qantivirus: Why is my antivirus/firewall flagging the Monero software i just downloaded as malware?
+  aantivirus: After you have downloaded the Monero software (GUI and CLI alike), may happen that your antivirus or firewall flags the executables as malware. Some antiviruses only warn you about the possible menace, others go as far as silently removing your downloaded wallet/daemon. This likely happens because of the integrated miner, which is used for mining and for @block verification. Some antiviruses may erroneously consider the miner as dangerous software and act to remove it.
```

Comment:
>After you have downloaded the Monero software (GUI and CLI alike), may happen that your antivirus ...

Maybe this would read better as

>After you have downloaded the Monero software (GUI and CLI alike), your antivirus or firewall ...

Reply:
You mean you would change to `your antivirus or firewall may flag the executable as malware`?

Reply:
Yes, just to get rid of the `may happen that` bit

Reply:
Done